CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -147,13 +63,8 @@ public void CanRemindLateDevelopersAtTenThirtyOnFriday()
             );
             
             _slackApi.Get("/api/users.list").Responds(slackGetUsersResponse);
-            
-            var slackPostMessageResponse = new SlackPostMessageResponse
-            {
-                IsOk = true
-            };
 
-            _slackApi.Post("/api/chat.postMessage").Responds(slackPostMessageResponse);
+            _slackApi.Post("/api/chat.postMessage").Responds("\"ok\": true");
```

Comment:
```suggestion
            _slackApi.Post("/api/chat.postMessage").Responds("{{ \"ok\": true }}");
```

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/9/commits/eb8ed423af0a29c0096eb16aeade0d16bf8a7ebd